app/controllers/idv_controller.rb

```diff
@@ -1,7 +1,30 @@
 class IdvController < ApplicationController
-  def p1
+  def register_user(args = nil)
+    agent = Proofer::Agent.new(vendor: :mock)
+    app_vars = params.slice(:first_name, :last_name, :dob)
+    applicant = Proofer::Applicant.new(app_vars)
+    session[:resolution] = agent.start(applicant)
+    session[:question_number] = 0
+    render 'question', locals: {quest: session[:resolution].questions[session[:question_number]]}
   end
```

Comment:
The proofer gem itself is pretty agnostic as to how various vendors work. It just knows how to load a particular vendor adapter and implements an abstract API that the adapter must implement.

@monfresh I'm curious what you mean by "response with a status to let the client know how to proceed" -- care to mock up some dummy code to illustrate what you mean?